acceptance-tests/dsl/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
   implementation project(':metrics:core')
   implementation project(':plugin-api')
   implementation project(':plugins:rocksdb')
+  implementation project(':plugins:securitymodule-bouncycastle')
```

Comment:
I know this is what I called it (roughly anyway) - but maybe I'm wrong - maybe its actually a "local-file security provider"?
I.e. people don't care that its backed by bouncy-castle, only that its created with a file.

Reply:
module has been renamed.